gradle/libs.versions.toml

```diff
@@ -125,6 +125,7 @@ slack-lint-compose = { module = "com.slack.lint.compose:compose-lint-checks", ve
 
 # KSP
 ksp-plugin = { module = "com.google.devtools.ksp:com.google.devtools.ksp.gradle.plugin", version = "1.9.10-1.0.13" }
+material = { group = "com.google.android.material", name = "material", version = "1.9.0" }
```

Comment:
In this [https://github.com/Ivy-Apps/ivy-wallet/pull/2748/files](https://github.com/Ivy-Apps/ivy-wallet/pull/2748/files) Date Picker is used instead of Time Picker which won't work. And as you said already Compose Material M3 is already a dependency added to the project and to use that here we may need to reconstruct some parts of the code. So, I believe using MDC here would be better here

Reply:
Okay, let's use your approach since it changes less code. I'm not very happy with adding more dependencies but it's fine for this case.